tests/pytests/integration/runners/test_nacl.py

```diff
@@ -0,0 +1,170 @@
+"""
+Tests for the nacl runner
+"""
+
+import pytest
+
+import salt.config
+import salt.utils.stringutils
+from tests.support.mock import patch
+
+try:
+    import salt.runners.nacl as nacl
+
+    HAS_PYNACL = True
+except (ImportError, OSError, AttributeError):
+    HAS_PYNACL = False
+
+pytestmark = [
+    pytest.mark.windows_whitelisted,
+]
+
+
+@pytest.mark.skipif(
+    not HAS_PYNACL, reason="skipping test_nacl, reason=PyNaCl is unavailable"
+)
+@pytest.fixture(scope="module")
```

Comment:
```suggestion
pytest.importorskip("libnacl")

pytestmark = [
    pytest.mark.windows_whitelisted,
]


@pytest.fixture(scope="module")
```

The runner imports `salt.utils.nacl` which doesn't trigger an `ImportError`